docs/en/integrations/elastic-package.asciidoc

```diff
@@ -0,0 +1,247 @@
+
+Use this command to create a new package or add more data streams.
+
+The command can help bootstrap the first draft of a package using embedded package template. It can be used to extend the package with more data streams.
+
+For details on how to create a new package, review the [HOWTO guide](https://github.com/elastic/elastic-package/blob/master/docs/howto/create_new_package.md).
+
+[discrete]
+=== `elastic-package export`
+
+_Context: package_
+
+Use this command to export assets relevant for the package, e.g. Kibana dashboards.
+
+[discrete]
+=== `elastic-package format`
+
+_Context: package_
+
+Use this command to format the package files.
+
+The formatter supports JSON and YAML format, and skips "ingest_pipeline" directories as it's hard to correctly format Handlebars template files. Formatted files are being overwritten.
+
+[discrete]
+=== `elastic-package install`
+
+_Context: package_
+
+Use this command to install the package in Kibana.
+
+The command uses Kibana API to install the package in Kibana. The package must be exposed via the Package Registry.
+
+[discrete]
+=== `elastic-package lint`
+
+_Context: package_
+
+Use this command to validate the contents of a package using the package specification (see: https://github.com/elastic/package-spec).
+
+The command ensures that the package is aligned with the package spec and the README file is up-to-date with its template (if present).
+
+[discrete]
+=== `elastic-package profiles`
+
+_Context: global_
+
+Use this command to add, remove, and manage multiple config profiles.
+
+Individual user profiles appear in ~/.elastic-package/stack, and contain all the config files needed by the "stack" subcommand.
+Once a new profile is created, it can be specified with the -p flag, or the ELASTIC_PACKAGE_PROFILE environment variable.
+User profiles are not overwritten on upgrade of elastic-stack, and can be freely modified to allow for different stack configs.
+
+[discrete]
+=== `elastic-package promote`
+
+_Context: global_
+
+Use this command to move packages between the snapshot, staging, and production stages of the package registry.
+
+This command is intended primarily for use by administrators.
+
+It allows for selecting packages for promotion and opens new pull requests to review changes. Please be aware that the tool checks out an in-memory Git repository and switches over branches (snapshot, staging and production), so it may take longer to promote a larger number of packages.
+
+[discrete]
+=== `elastic-package publish`
+
+_Context: package_
+
+Use this command to publish a new package revision.
+
+The command checks if the package hasn't been already published (whether it's present in snapshot/staging/production branch or open as pull request). If the package revision hasn't been published, it will open a new pull request.
+
+[discrete]
+=== `elastic-package service`
+
+_Context: package_
+
+Use this command to boot up the service stack that can be observed with the package.
```

Comment:
It would be nice to present an example somewhere in docs. For example: boot up "nginx" service image to develop new dashboards.